sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/Conditions.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration.implementation;
```

Comment:
So, with this implementation, the only way to add a new unknown condition is through modifying the `value` field which is a string of json. Doesn't seem super useful. And could easily result in json format issues.

Reply:
Yes. With the current design, The only way you can add a new unknown condition is through `value` or through the Azure AppConfig portal. We are trying to align with what spec schema has. Not introduce `extended unknown` property.
I think that is what .NET ships with. Might be worth having a deeper discussion on how to support the `extended unknown` property in swagger?

Reply:
I am fine with how this is, just wanted to bring up the topic.